src/components/molecules/TodoInputTask.vue

```diff
@@ -0,0 +1,71 @@
+<script setup lang="ts">
+import useVuelidate from '@vuelidate/core'
+import { maxLength, required } from '@vuelidate/validators'
+import { computed, ref } from 'vue'
+
+import TodoInput from '@/components/atoms/TodoInput.vue'
+import TaskStatusIcon from '@/components/molecules/TaskStatusIcon.vue'
+import { app } from '@/constants/constants'
+import { useTodoTasksStore } from '@/stores/todoTasksStore'
+
+interface TodoInputTaskProps {
+  isTaskCompleted: boolean
+}
+
+withDefaults(defineProps<TodoInputTaskProps>(), {
+  isTaskCompleted: false
+})
+const todoTasksStore = useTodoTasksStore()
+
+const inputTodoTaskValue = ref<string>('')
+const rules = computed(() => ({
+  inputTodoTaskValue: {
+    required,
+    maxLength: maxLength(app.validation.inputTask.length.max)
+  }
+}))
+const $v = useVuelidate(rules, { inputTodoTaskValue })
+
+const addNewTask = () => {
+  const result = $v.value.$validate()
+  result
+    .then((res) => {
+      if (res) {
+        todoTasksStore.addTask(inputTodoTaskValue.value)
+        inputTodoTaskValue.value = ''
+        $v.value.$reset()
+      }
+    })
+    .catch((err) => {
+      console.log('An error occurred', err)
+    })
+}
+</script>
+
+<template>
+  <div class="flex flex-col justify-start">
+    <div
+      class="flex flex-row justify-between p-2 bg-white rounded-lg input-wrapper"
+      :class="{
+        'border border-pink-500': $v.inputTodoTaskValue.$error && $v.inputTodoTaskValue.$dirty
```

Comment:
better to create computed prop for this value

Reply:
in computed you can use $v.value.inputTodoTaskValue.$dirty and $v.value.inputTodoTaskValue.$error